src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -874,6 +915,14 @@ public void AddRequestContext()
                 _parameters.Add(KnownParameters.RequestContext);
             }
 
+            private void AddRequestParameterWithMediaTypes(RequestParameter requestParameter, ICollection<string> requestMediaTypes)
+            {
+                var parameter = _parent.BuildContentTypeParameterWithMediaTypes(requestParameter, requestMediaTypes);
```

Comment:
`BuildContentTypeParameterWithMediaTypes` can be declared in `RequestMethodParametersBuilder`

Reply:
Even better, you can just inline it here.